File: FLUTTER/igest_app/lib/models/principal/provincia.dart
class Provincia {
  String idProvincia;
  String idDepartamento;
  String nomProvincia;

  Provincia({
    required this.idProvincia,
    required this.idDepartamento,
    required this.nomProvincia,
  });

  factory Provincia.fromJson(Map<String, dynamic> json) {
    return Provincia(
      idProvincia: json['id_provincia'],
      idDepartamento: json['id_departamento'],
      nomProvincia: json['nom_provincia'],
    );
  }

  Map<String, dynamic> toJson() {
    return {
      'id_provincia': idProvincia,
      'id_departamento': idDepartamento,
      'nom_provincia': nomProvincia,
    };
  }
}

File: SPRING/igest.server.model/src/main/java/com/yupanastudio/igest/server/principal/dml/Provincia.java
/*
 * Document   : Provincia
 * Author     : Franklin Barzola Olivares <frankoantonio@gmail.com>
 */
package com.yupanastudio.igest.server.principal.dml;

import org.springframework.data.annotation.Id;

public class Provincia {

    @Id
    private String idProvincia;
    private String idDepartamento;
    private String nomProvincia;

    public String getIdProvincia() {
        return idProvincia;
    }

    public void setIdProvincia(String idProvincia) {
        this.idProvincia = idProvincia;
    }

    public String getIdDepartamento() {
        return idDepartamento;
    }

    public void setIdDepartamento(String idDepartamento) {
        this.idDepartamento = idDepartamento;
    }

    public String getNomProvincia() {
        return nomProvincia;
    }

    public void setNomProvincia(String nomProvincia) {
        this.nomProvincia = nomProvincia;
    }

}
